core/src/group16/project/game/views/GameScreen.kt

```diff
@@ -189,6 +205,10 @@ class GameScreen(val gameController: StarBattle, val fbic: FirebaseInterface) :
             bbar.setPosition((stage.width - 150f)*i - 20f, padding - 20f)
             stage.addActor(bbar)
 
+            var lobbycode = VisLabel(GameInfo.currentGame)

```

Comment:
Very good stuff!
The thought of having the bottombar first was to place the different "skills"/"powerups" in the future update, so the player can easily access and toggle them. Therefore, moving this to another spot early on while you are at it is a good idea too. :)

See attached image:
![image](https://user-images.githubusercontent.com/20891306/164118129-191365cc-9afe-4dce-902b-a07186ca1b5e.png)

Reply:
where should I move it?

Reply:
Maybe at the top? with the gamestate status bar